docs/copilot/copilot-vscode-features.md

```diff
@@ -127,7 +127,9 @@ As you're coding in the editor, you can use Copilot to generate code completions
 
 ## Customize AI code generation
 
-Copilot can generate responses that match the coding style, tools, and developer workflow of your team or project, provided it has the right context. You can provide custom instructions to help Copilot adjust to your preferences, so that you don't have to provide these details every time you make a Copilot request. These custom instructions are automatically added to your requests. Get more information about [customizing Copilot in VS Code](/docs/copilot/copilot-customization.md).
+Define [custom instructions](/docs/copilot/copilot-customization.md#custom-instructions) to help Copilot generate code or review code that matches the coding style, tools, and developer workflow of your team or project.
+
+With reusable prompt files, you can specify common prompt instructions and relevant content in a Markdown file, that you can then reuse in your chat prompts.
```

Comment:
Add the file extension for clarity? Otherwise folks can try to use usual .md files and get confused - this happened even with ppl on our team 🤗 

```suggestion
With reusable prompt files, you can specify common prompt instructions and relevant content in a Markdown file (`*.prompt.md`), that you can then reuse in your chat prompts.
```